### ISSUE TITLE
[Bug]: Govee H5182 Data not parsing / no data

### ISSUE THREAD

**Issue by phdonnelly** [custom-components/ble_monitor#914]
### What happened?

Got a new H5182 from amazon; supported recently via Issue 851. It works great in the govee app, but wasn't autodetected by the add-on like my other govee devices; I added the mac address manually but am still not getting any data; I turned on logging and put the output below.

### Sensor type

Govee H5182

### Relevant log output

```shell
H5182

2022-07-01 18:54:29 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865e24ffff860898ffffb5
2022-07-01 18:54:33 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401866018ffff860898ffffb5
2022-07-01 18:54:35 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018660e0ffff860898ffffb4
2022-07-01 18:54:35 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb4
2022-07-01 18:54:39 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018661a8ffff860898ffffad
2022-07-01 18:54:40 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018661a8ffff860898ffffaf
2022-07-01 18:54:40 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018661a8ffff860898ffffb9
2022-07-01 18:54:41 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018661a8ffff860898ffffad
2022-07-01 18:54:44 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018660e0ffff860898ffffb8
2022-07-01 18:54:45 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401866018ffff860898ffffac
2022-07-01 18:54:46 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865fb4ffff860898ffffb2
2022-07-01 18:54:47 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865f50ffff860898ffffb4
2022-07-01 18:54:49 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865e24ffff860898ffffb4
2022-07-01 18:54:50 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865e24ffff860898ffffad
2022-07-01 18:54:50 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cae
2022-07-01 18:54:51 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865d5cffff860898ffffb4
2022-07-01 18:54:52 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865c30ffff860898ffffad
2022-07-01 18:54:53 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865c30ffff860898ffffab
2022-07-01 18:54:54 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865b68ffff860898ffffb2
2022-07-01 18:54:59 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865a3cffff860898ffffb4
2022-07-01 18:55:02 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018658acffff860898ffffb6
2022-07-01 18:55:02 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb6
2022-07-01 18:55:03 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018658acffff860898ffffaa
2022-07-01 18:55:05 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018657e4ffff860898ffffb2
2022-07-01 18:55:06 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865780ffff860898ffffad
2022-07-01 18:55:10 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865654ffff860898ffffae
2022-07-01 18:55:10 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cae
2022-07-01 18:55:11 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865654ffff860898ffffb2
2022-07-01 18:55:15 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018654c4ffff860898ffffb5
2022-07-01 18:55:15 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb6
2022-07-01 18:55:16 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018653fcffff860898ffffb2
2022-07-01 18:55:16 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb2
2022-07-01 18:55:17 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018653fcffff860898ffffb5
2022-07-01 18:55:17 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb6
2022-07-01 18:55:19 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865398ffff860898ffffb0
2022-07-01 18:55:19 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb1
2022-07-01 18:55:20 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865334ffff860898ffffaa
2022-07-01 18:55:21 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e40186526cffff860898ffffb0
2022-07-01 18:55:21 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb0
2022-07-01 18:55:23 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e40186526cffff860898ffffb3
2022-07-01 18:55:24 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e40186526cffff860898ffffb1
2022-07-01 18:55:25 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e40186526cffff860898ffffb0
2022-07-01 18:55:26 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e40186526cffff860898ffffad
2022-07-01 18:55:26 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cad
2022-07-01 18:55:27 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e40186526cffff860898ffffb1
2022-07-01 18:55:27 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb1
2022-07-01 18:55:29 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865208ffff860898ffffa6
2022-07-01 18:55:30 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018651a4ffff860898ffffb3
2022-07-01 18:55:31 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018651a4ffff860898ffffad
2022-07-01 18:55:34 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865140ffff860898ffffb1
2022-07-01 18:55:36 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865140ffff860898ffffb1
2022-07-01 18:55:37 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018650dcffff860898ffffa5
2022-07-01 18:55:37 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0ca6
2022-07-01 18:55:38 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018650dcffff860898ffffae
2022-07-01 18:55:38 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cae
2022-07-01 18:55:39 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018650dcffff860898ffffa5
2022-07-01 18:55:39 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0ca7
2022-07-01 18:55:40 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e4018650dcffff860898ffffad
2022-07-01 18:55:41 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865014ffff860898ffffa5
2022-07-01 18:55:41 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0ca6
2022-07-01 18:55:42 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401865014ffff860898ffffae
2022-07-01 18:55:46 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864fb0ffff860898ffffad
2022-07-01 18:55:47 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864fb0ffff860898ffffa4
2022-07-01 18:55:47 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0ca4
2022-07-01 18:55:48 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864fb0ffff860898ffffad
2022-07-01 18:55:50 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864fb0ffff860898ffffb2
2022-07-01 18:55:52 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864f4cffff860898ffffad
2022-07-01 18:55:55 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864f4cffff860898ffffb2
2022-07-01 18:55:55 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864ee8ffff860898ffffae
2022-07-01 18:55:56 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864ee8ffff860898ffffb1
2022-07-01 18:55:56 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb2
2022-07-01 18:55:59 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864e84ffff860898ffffae
2022-07-01 18:56:00 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864e20ffff860898ffffad
2022-07-01 18:56:00 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0caf
2022-07-01 18:56:03 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864dbcffff860898ffffb1
2022-07-01 18:56:05 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864dbcffff860898ffffb2
2022-07-01 18:56:07 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864d58ffff860898ffffb2
2022-07-01 18:56:09 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864d58ffff860898ffffb2
2022-07-01 18:56:10 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864cf4ffff860898ffffae
2022-07-01 18:56:11 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864cf4ffff860898ffffb4
2022-07-01 18:56:11 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cb2
2022-07-01 18:56:13 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864c90ffff860898ffffb5
2022-07-01 18:56:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864c90ffff860898ffffae
2022-07-01 18:56:14 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cae
2022-07-01 18:56:16 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864c2cffff860898ffffae
2022-07-01 18:56:16 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e27020104016530303031ce1b1aff4c000215494e54454c4c495f524f434b535f48575075f2ff0cae
2022-07-01 18:56:17 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864c2cffff860898ffffb5
2022-07-01 18:56:19 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864bc8ffff860898ffffb1
2022-07-01 18:56:20 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864bc8ffff860898ffffad
2022-07-01 18:56:22 INFO (Thread-3) [custom_components.ble_monitor.ble_parser] BLE advertisement received from MAC/UUID ce3130303065: 043e28020100016530303031ce1c0201060303518214ff30306501000101e401864b64ffff860898ffffb0
```

**Comment by Ernst79** [comment]
I just did a test, and can confirm the issue. I will look into it and make a fix.

**Comment by Ernst79** [comment]
Fixed in 8.9.6
